src/hotspot/share/compiler/compiler_globals.hpp

```diff
@@ -382,7 +392,20 @@
           "If compilation is stopped with an error, capture diagnostic "    \
           "information at the bailout point")                               \
                                                                             \
-
+  /* flags to control training and deployment modes  */                     \
```

Comment:
I think flags should include `AOT` to be clear what they are used for. For example, `RecordAOTTraining`.

Reply:
Consider moving these flags to `cds_globals.hpp` since we will not use this without CDS. I am putting AOT code flags there.

Reply:
@iklam asked to move it to compiler. :)